lib/media/streaming_engine.js

```diff
@@ -818,6 +838,7 @@ shaka.media.StreamingEngine = class {
       stream,
       type: stream.type,
       segmentIterator: null,
+      segmentPrefetch: this.createSegmentPrefetch_(stream),
```

Comment:
We probably should only make prefetchers for video and audio streams, not text or image.

Reply:
updated createSegmentPrefetch_ to return null if stream.type is not VIDEO nor AUDIO